### ISSUE TITLE
Rename few go files under command group dir

### ISSUE THREAD

**Issue by navidshaikh** [knative/client#229]
This is probably the left over from issue https://github.com/knative/client/issues/66

Looking at how go files are structured in repo, each command group is a dir and leaf command files has an extra word for eg `service_create.go`, `service_list.go`. We should rename, for eg:
`pkg/kn/commands/service/service_create.go` - > `pkg/kn/commands/service/create.go `

```
├── service
│   ├── configuration_edit_flags.go
│   ├── human_readable_flags.go
│   ├── service_create.go
│   ├── service_create_test.go
│   ├── service_delete.go
│   ├── service_describe.go
│   ├── service_describe_test.go
│   ├── service.go
│   ├── service_list_flags.go
│   ├── service_list.go
│   ├── service_list_test.go
│   ├── service_update.go
│   ├── service_update_test.go
│   └── wait_args.go
```

**Comment by rhuss** [comment]
Same is partially true for the files within `revision/`

**Comment by zhanggbj** [comment]
Also updated `plugin/`

**Comment by navidshaikh** [comment]
Fixed by #245
